packages/components/link/src/link.tsx

```diff
@@ -0,0 +1,143 @@
+import type { Theme } from '@emotion/react';
+import type { MessageDescriptor } from 'react-intl';
+import React from 'react';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import { css, useTheme } from '@emotion/react';
+import { FormattedMessage } from 'react-intl';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+import { filterInvalidAttributes, warning } from '@commercetools-uikit/utils';
+
+type TExtendedTheme = Theme & {
+  [key: string]: string;
+};
+type TLinkShape = {
+  pathname: string;
+  search: string;
+  hash?: string;
+  state: unknown;
+};
+
+type TLinkProps = {
+  tone?: 'primary' | 'inverted';
+  // @TODO: deprecate?
```

Comment:
I just realised that this will be covered with V12.
Setting this PR to `on hold` in the mean time